subprojects/common/junit-utils/src/main/java/ru/avito/util/RandomUtils.kt

```diff
@@ -184,6 +186,15 @@ fun randomPhone(): String {
     return "+7" + randomString(10, Characters.Digits)
 }
 
+@Deprecated("start and end could be negative", replaceWith = ReplaceWith("randomDate(start, end)"))
 fun randomDate(start: Long, end: Long): Date {
     return Date(randomLong(start, end))
 }
+
+fun randomDate(start: Date, end: Date): Date {
+    return Date(randomLong(start.time, end.time))
+}
+
+fun randomDate(): Date = Date(randomLong())
+
+fun randomLongDate(): Long = Date(randomLong()).time
```

Comment:
```suggestion
fun randomTimestamp(): Long = Date(randomLong()).time
```
the `timestamp` is a well known term.

OR

```suggestion
fun randomDateInMillis(): Long = Date(randomLong()).time
```
to be precise.

Reply:
Done